vscode-extension/editor/src/VSCodeEditor.tsx

```diff
@@ -233,6 +231,15 @@ export default function VSCodeEditor() {
     [aiConfigServerUrl, vscode]
   );
 
+  const deleteModelSettings = useCallback(
```

Comment:
nit: Why is this called `deleteModelSettings` instead of `deleteModel`?

Reply:
I called the callback deleteModelSettings because that's a much clearer user-facing name. IMO `delete_model` is not clear in aiconfig sdk right now since I actually would have interpreted it as deleting the model parser if I didn't look through the code